src/toil/jobStores/googleJobStore.py

```diff
@@ -179,10 +188,28 @@ def assignID(self, jobDescription):
                   jobStoreID, '<no command>' if jobDescription.command is None else jobDescription.command)
         jobDescription.jobStoreID = jobStoreID
 
+    jobsPerBatchInsert = 200
```

Comment:
We want all our new variables (and eventually all our old ones) to be `snake_case`. So this should be called `jobs_per_batch_insert` instead.

Reply:
And if it's just going to be a constant, it should be `JOBS_PER_BATCH_INSERT` and probably live at the top of the module under the imports.

And despite its informative variable name, it should have a comment explaining what it controls, and with some justification as to why it ought to be 200 and not, say, 20 or 2000.